src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ConfigurationGeneral.xaml

```diff
@@ -50,6 +50,11 @@
             <DataSource Persistence="ProjectFileWithInterception" PersistedName="TargetFrameworkMonikers" HasConfigurationCondition="False" SourceOfDefaultValue="AfterContext"/>
         </StringProperty.DataSource>
     </StringProperty>
+    <BoolProperty Name="AutoGenerateBindingRedirects" DisplayName="Auto-Generate Binding Redirects">
```

Comment:
This shouldn't be needed. We don't consume this value anywhere except in the browse object.

Reply:
OK. I'll remove this one and double check that everything still work.